weblate/api/views.py

```diff
@@ -263,17 +264,30 @@ def repository(self, request, **kwargs):
         return Response(data)
 
 
+class UserFilter(filters.FilterSet):
+    username = filters.CharFilter(field_name="username", lookup_expr="startswith")
+
+    class Meta:
+        model = User
+        fields = ["username"]
+
+
 class UserViewSet(viewsets.ModelViewSet):
     """Users API."""
 
     queryset = User.objects.none()
-    serializer_class = UserSerializer
+    serializer_class = FullUserSerializer
```

Comment:
Does it have to be defined here? I'd rather just have the get_serializer_class method

Reply:
Yeah. Not exactly needed.